posts/MigueTripi/01.isContractIssue.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Using OpenZepellin standard IsContract() could be unsafe'
+published: false
+description: We are at risk if we do not use this consciously
+tags: 'Solidity, EVM, Etherium, hack'
+cover_image: ./assets/01.cover.png
```

Comment:
La cover image ideal para un post según [la guía de dev.to](https://dev.to/p/editor_guide) es de 1000x420. No se cómo se verá esta imagen cuadrada.

Reply:
Done!